drivers/wifi/infineon/airoc_wifi.c

```diff
@@ -761,7 +761,7 @@ static const struct wifi_mgmt_ops airoc_wifi_mgmt = {
 	.disconnect = airoc_mgmt_disconnect,
 	.ap_enable = airoc_mgmt_ap_enable,
 	.ap_disable = airoc_mgmt_ap_disable,
-#if defined(CONFIG_NET_STATISTICS_ETHERNET)
+#if defined(CONFIG_NET_STATISTICS_WIFI)
```

Comment:
Could you split this change to different commit as the changes are not related to each other at all.

Reply:
done